XR_APILAYER_NOVENDOR_toolkit/menu.cpp

```diff
@@ -1415,6 +1415,16 @@ namespace {
                  }});
             m_menuEntries.back().acceleration = 10;
 
+            m_menuEntries.push_back({MenuIndent::OptionIndent,
+                                     "No inter eye rotation workaround",
```

Comment:
I'm trying to think of a better name for this so it's more obvious. Pimax calls it parallel projection, so this might be a more common name